src/Command/BakeCommand.php

```diff
@@ -164,4 +164,27 @@ protected function deleteEmptyFile(string $path, ConsoleIo $io): void
             $io->out(sprintf('<success>Deleted</success> `%s`', $path), 1, ConsoleIo::NORMAL);
         }
     }
+
+    /**
+     * The Regex used here basically states that:
```

Comment:
Please add a summary line at the top.

Reply:
I didnt think u would copy paste 1:1 :)
This could need a bit fine tuning

Reply:
done 👍🏻

Reply:
@dereuromark I like to be straight forward 😛